### DISCUSSION TITLE
Add text field to number of item in cart

### DISCUSSION THREAD

**Discussion by ghost** [jonassiewertsen/statamic-butik#170] [Ideas]
Clicking the + or - buttons when dealing with a large volume is tedious.

Having the option for user to type the value of number of items to add to cart, or remove from cart, would be ideal.

**Comment by jonassiewertsen** [comment]
> Clicking the + or - buttons when dealing with a large volume is tedious.
> 
> Having the option for user to type the value of number of items to add to cart, or remove from cart, would be ideal.

I agree. This is definitely a feature we do need to make possible.
